src/test/docker/docker-compose.yml

```diff
@@ -3,6 +3,11 @@ services:
   dropwizard:
     build: ./docker-dropwizard-example/
     container_name: dropwizard
+    image: example/dropwizard
```

Comment:
Only one of build and image should be used in a `docker-compose.yml` file, so this image line shouldn't be here.